netDumbster.Test/TestsBase.cs

```diff
@@ -254,20 +269,17 @@ public void Send_One_Mail()
         }
 
         [Fact]
-        public void If_Client_Is_Not_Disposed_Server_Everything_Keeps_Working()
+        public void If_Client_Is_Not_Disposed_Server_Keeps_Working()
         {
-            var port = 25;
             var host = "localhost";
-            using (SimpleSmtpServer emailServer = this.StartServer(port))
-            {
-                SmtpClient client = new SmtpClient(host, port);
-                client.Send("noone@nowhere.com", "nobody@nowhere.com", "This is an email", "body of email");
-            }
+            using var emailServer = Configuration.Configure().WithRandomPort().Build();
+
+            var client = new SmtpClient(host, emailServer.Configuration.Port);
+            client.Send("noone@nowhere.com", "nobody@nowhere.com", "This is an email", "body of email");
 
-            using (SimpleSmtpServer emailServer = this.StartServer(port))
+            using (var client2 = new SmtpClient(host, emailServer.Configuration.Port))

```

Comment:
@cmendible I had to change this test. While running on unix machines, it was throwing a socket exception. I couldn't debug much so I don't know exactly what's the problem. But.. looking at the test name it seems I could re-write it the way I did. The question is: Does this makes sense or we do need to create the server twice?

Reply:
On unix machines we'll have to see figure out what the problem is. Perhaps we need to open a new issue to fix before closing this PR. That c
test is in place cause of this bug: #19